work/check_for_placeholders.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+
+from lib.utils import *
+
+def run(site_id: str, org_id: str, app: dict):
+    logging.info('Replace placeholders with multimedia links for org_id: {}'.format(org_id))
+    site_folder = APP['archive_folder']
+
+    xml_src = r'{}{}-archive/lessonbuilder.xml'.format(site_folder, site_id)
+    xml_old = r'{}{}-archive/lessonbuilder.old'.format(site_folder, site_id)
+    shutil.copyfile(xml_src, xml_old)
+
+    remove_unwanted_characters(xml_src)
+
+    file_path = os.path.join(site_folder, xml_src)
+
+    with open(file_path, "r", encoding="utf8") as fp:
+        soup = BeautifulSoup(fp, 'xml')
+        pages = soup.find_all('page')
```

Comment:
rename this and pages below to refer explicitly to what they are. Re-using a var name within the same scope is definitely confusing.